.circleci/real_config.yml

```diff
@@ -5578,6 +5641,9 @@ workflows:
               # enroot import docker://determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-gpu-24586f0.sqsh
               # enroot create /lustre/ssd/foundation_engineering/determinedai+environments+cuda-11.3-pytorch-1.10-tf-2.8-gpu-24586f0.sqsh
               # image: determinedai+environments+cuda-11.3-pytorch-1.10-tf-2.8-gpu-24586f0
+              image:
+                cpu: determinedai/pytorch-tensorflow-cpu-dev:e960eae
```

Comment:
I think you can replace this and the other config with the pipeline parameter `<<pipeline.parameters.default-pt-tf-cpu-hpc-image>>`. It would be nice having this defined in only one spot.